src/main/java/org/fundacionjala/pivotal/common/PropertiesConfig.java

```diff
@@ -0,0 +1,61 @@
+package org.fundacionjala.pivotal.common;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * PropertiesConfig is class for read config.properties.
+ */
+public final class PropertiesConfig {
+    /**
+     * properties.
+     */
+    private Properties properties;
+    /**
+     * PropertiesConfig.
+     */
+    private static PropertiesConfig propertiesConfig;
+    /**
+     * inputStream.
+     */
+    private InputStream inputStream;
+
+    /**
+     * @return a instance of PropertiesConfig.
+     */
+    public static PropertiesConfig getPropertiesConfig() {
+        if (propertiesConfig == null) {
+            propertiesConfig = new PropertiesConfig();
+        }
+        return propertiesConfig;
+    }
+
+    /**
+     * This method initialize properties and input stream.
+     */
+    private PropertiesConfig() {
+        try {
+            properties = new Properties();
+            inputStream = new FileInputStream("config.properties");
+            properties.load(inputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)